### ISSUE TITLE
Parsing Failed, can't find Eigen file

### ISSUE THREAD

**Issue by PhiliptHdL** [Tudat/tudat#597]
Hello, 

I am part of the Numerical Astrodynamics class who has just started assignment 2. Due to a computer crash I had to reinstall Tudat on another computer and am now getting an error that when I try to run the CMakeLists.txt file provided for that assignment that it cant find the Eigen3 files. Now I have rerun the inital CMakeLists.txt that I used for assignment 1 and ran the code of assignment 1 which is working as it should and I am getting the expected output files. Here I have the General Messages of when I try to open the project for assignment 2. 
[Tudat Eigen Parsing Error.txt](https://github.com/Tudat/tudat/files/3951389/Tudat.Eigen.Parsing.Error.txt). 

Thank you, 

Philip

**Comment by DominicDirkx** [comment]
Hi Philip,

Before we check the code for your assignment: did Tudat (without teh applications) compile correctly, and did the unit tests run?

For your error in assignment 2, have a look at the Brightspace forum:

https://brightspace.tudelft.nl/d2l/le/192056/discussions/threads/31866/View

This may be the source of your error as well,

Dominic

**Comment by PhiliptHdL** [comment]
Hello Dominic, 

Yes so the compile showed I think only yellow errors which I was told to ignor by the instructions and the unit tests all passes. 

Although that does show something I missed in the tudatbundle/CMakeLists.txt file this is not the project we open for assignment 2 right. We will be running assignment 2 code using the src2/CMakeLists.txt file which is the one failing due to a problem with not being able to find eigen files. You also say there I should not change the src2/CMakeLists.txt file so I don't see how that change would help if we are making a change to the tudatbundle/CMakeLists.txt file while running the src2/CMakeLists.txt file

Cheers, 

Philip

**Comment by DominicDirkx** [comment]
No, you **always** open the tudatBundle/CMakeLists.txt to run tudat applications or tests.

**Comment by PhiliptHdL** [comment]
Alright, I have added the line to the tudatBundle/CMakeLists.txt that is "add_subdirectory( "${PROJECTROOT}/tudatApplications/NumericalAstrodynamics2019_PhiliptenHoutedeLange\src2")" and am now getting a new parsing failure with an error about an "Invalid escape sequence". Here is the output I am getting. 
[Tudat Parsing Error 2.txt](https://github.com/Tudat/tudat/files/3955267/Tudat.Parsing.Error.2.txt)

I also ran it with the file path without the src2 but then it could not find the CMakeLists.txt in that location. 

Thanks already though for you help and patience with my nonunderstanding of how to set this up. 

Cheers, 

Philip
